IDE/IAR-EWARM/embOS/user_settings.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _EMBOS_USER_SETTINGS_H_
+#define _EMBOS_USER_SETTINGS_H_
+
```

Comment:
The IDE files and README need to go into the pull request.  This isn't a one off but an addition that should be useful to others as well.